Schedule-API/Schedule-API/obj/ScheduleAPI.csproj.nuget.g.props

```diff
@@ -3,9 +3,9 @@
   <PropertyGroup Condition=" '$(ExcludeRestorePackageImports)' != 'true' ">
```

Comment:
@zynori I think this file should not be tracked by `git`.
https://stackoverflow.com/questions/43069745/should-nuget-props-and-targets-files-be-part-of-the-source-code-repository 
something might can helo you :)